apps/air-discount-scheme/web/screens/Admin/Admin.tsx

```diff
@@ -92,6 +93,48 @@ const Admin: Screen = ({}) => {
     setFilters(data)
   }
 
+  const downloadCSV = () => {
```

Comment:
Would like to see this function in e.g.:
```typescript
import { downloadCSV } from './utils'
```
Where utils is in:
`apps/air-discount-scheme/web/screens/Admin/utils.ts`